templates/configfile.j2

```diff
@@ -50,7 +50,7 @@ install_hostagent={{ sap_hana_deployment_deploy_hostagent }}
 db_isolation=low
 
 # Create initial tenant database ( Default: y )
-create_initial_tenant=y
+create_initial_tenant={{ sap_hana_deployment_create_initial_tenant }}
```

Comment:
I think this should be defaulted to `y` since this is a new variable, has no default value and will break backwards compatibility

Reply:
I agree, it is defaulted to `y`now